<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StampSelection</class>
 <widget class="QWidget" name="StampSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="yellowSpritePushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/stamp/Resources/stamp/yellowSprite.png</normaloff>:/Resources/stamp/Resources/stamp/yellowSprite.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="blueSpritePushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/stamp/Resources/stamp/blueSprite.png</normaloff>:/Resources/stamp/Resources/stamp/blueSprite.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="warriorSpritePushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/stamp/Resources/stamp/warriorSprite.png</normaloff>:/Resources/stamp/Resources/stamp/warriorSprite.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="crabSpritePushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/stamp/Resources/stamp/crabSprite.png</normaloff>:/Resources/stamp/Resources/stamp/crabSprite.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>yellowSpritePushButton</sender>
   <signal>clicked()</signal>
   <receiver>StampSelection</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSpritePushButton</sender>
   <signal>clicked()</signal>
   <receiver>StampSelection</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>warriorSpritePushButton</sender>
   <signal>clicked()</signal>
   <receiver>StampSelection</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>crabSpritePushButton</sender>
   <signal>clicked()</signal>
   <receiver>StampSelection</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
